funwithphysics/src/Components/Home/Home.js

```diff
@@ -25,12 +25,12 @@ const Home = () => {
 			<Footer />
 			{/* <!-- Back to top button --> */}
 			<button className="gotopbtn" onClick={scroll}> <i className="fas fa-arrow-up"></i> </button>
-		</React.Fragment>
+		</>
```

Comment:
Why did u remove React Fragments?

Reply:
</> this also works thatswhy

Reply:
See first of all u said it's not working on ur PC and then when u r changing something how would u know whether it is working or not?

Reply:
sir close this